core/.js/src/main/scala/io/udash/bindings/modifiers/PropertyModifier.scala

```diff
@@ -11,21 +11,11 @@ private[bindings] class PropertyModifier[T](
   override val customElementsReplace: DOMManipulator.ReplaceMethod
 ) extends ValueModifier[T] {
 
-  def this(property: ReadableProperty[T], builder: T => Seq[Node], checkNull: Boolean) = {
-    this(property, (data: T, _: Binding.NestedInterceptor) => builder(data), checkNull, DOMManipulator.DefaultElementReplace)
+  def this(property: ReadableProperty[T], builder: T => Seq[Node], checkNull: Boolean, customElementReplace: DOMManipulator.ReplaceMethod) = {
```

Comment:
It's `Elements` in bindings and `Element` here - let's use one (probably the one from bindings as it has way more external usage).